mentor.html

```diff
@@ -3,40 +3,150 @@
-        <hr class="section-heading-spacer">
+      <div class="col-md-8 col-md-offset-2">
+        <div class="clearfix"></div>
+        <h2 class="section-heading">
+          You want to know what it means to mentor a new contributor to our project?
+          Wonderful, let us guide you on what we think is important to know, what
+          our best practices are and which behavior in mentors we desire.
+          Here is our ten step program for openSUSE 101!
+        </h2>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section id="mentor">
+  <div class="container" id="projects">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
         <div class="clearfix"></div>
-        <h2 class="section-heading">What makes a good mentor?</h2>
+        <h3>1. Mentees First</h3>
+        <p>
+          We believe in putting the mentees needs first. You need to emphasize
+          with the mentee, understand his personality, requirements and
+          circumstances. Make an effort to understand where the mentee comes from
+          and what you need to do, to get them to contribute to free software.
+          Mentoring is about them, not about your or openSUSEs needs.
+        </p>
+        <h3>2. Build a Relationship and have a lot of fun...</h3>
+        <p>
+          The most important goal is that you build a relationship based on trust
+          and respect with the mentee and that you have fun together. Without this
+          everything else you'll do is in vein! Such relationships do not just
+          happen, they need ongoing effort and monitoring, especially during the
+          early stages.
+        </p>
+        <h3>3. Be a role model</h3>
+        <p>
+          Today this sounds like a weird thing from the 60s and it makes most
+          people uncomfortable but you need to be a role model for your mentee!
+          Free Software development is as much about behavior in a community,
+          knowing what you want and the ability to convey this to others than it
+          is about pointers, gcc macros and git command line options. You need
+          to ensure that your mentee can follow your example and that you're
+          not passing on your bad habits. You are responsible for developing
+          the competence AND character of the mentee!
+        </p>
+        <h3>4. Plan</h3>
+        <p>
+          The first thing you'll do together with the mentee is to prepare for the
+          far future. Precise, reasonable and well-thought-out planing is a must
+          for every successful mentoring project. Getting into free software
+          development can be an overwhelming and daunting endeavor. A decent
+          plan provides a path and helps to take one step after the other into
+          this strange new world. It's your duty that the plan the mentee comes
+          up with, is do-able, provides orientation and is technically sound.
+          You're the expert!
```

Comment:
Planning can be difficult as it's impossible to lay out a plan which stays valid over the course of the whole project. I would rather say "Set goals and expectation". These provide directions and define how you are following them. The concrete plan how to do things will be much more short term and change with every step.